hibernate-reactive-core/build.gradle

```diff
@@ -48,3 +48,18 @@ test {
   systemProperty 'docker', project.hasProperty('docker') ? 'true' : 'false'
   systemProperty 'db', project.hasProperty('db') ? project.getProperty('db') : 'postgresql'
 }
+
+spotless {
+    //Don't fail during the check: rather than enforcing guidelines, we use this plugin to fix mistakes automatically.
+    enforceCheck false
+    java {
+        licenseHeaderFile 'spotless.license.java'
+        //These seem rather cool but let's introduce them later iff there's consensus - and probably at a better time.
+//        removeUnusedImports()
+//        trimTrailingWhitespace()
+//        indentWithTabs()
```

Comment:
Don't enable this one. It does some sort of weird violence to my Javadoc comments.

Reply:
right I noticed the same. I guess we can explore these options later: I also wouldn't want it to reformat the whole project right now, it would cause a load of conflicts with all current work in progress.